src/scales/scale.category.js

```diff
@@ -53,10 +53,25 @@ module.exports = function() {
 
 		getLabelForIndex: function(index, datasetIndex) {
 			var me = this;
-			var data = me.chart.data;
+			var chart = me.chart;
+			var data = chart.data;
 			var isHorizontal = me.isHorizontal();
 
-			if (data.yLabels && !isHorizontal) {
```

Comment:
Don't we still need to check for `data.yLabels`?

Reply:
I've created a pen with code from this PR and sample charts from https://github.com/chartjs/Chart.js/pull/2849 and https://github.com/chartjs/Chart.js/pull/3649 related to this line of code / yLabels.
https://codepen.io/kurkle/full/dweMZB